app/src/main/java/de/xikolo/utils/FileProviderUtil.kt

```diff
@@ -0,0 +1,18 @@
+package de.xikolo.utils
+
+import android.net.Uri
+import androidx.core.content.FileProvider
+import de.xikolo.App
+import de.xikolo.BuildConfig
+import java.io.File
+
+class FileProviderUtil : FileProvider() {
+
+    companion object {
+
```

Comment:
Please remove the empty line since there is also no after the closing bracket.

Reply:
Fixed